front/components/assistant/AssistantDetailsDropdownMenu.tsx

```diff
@@ -153,13 +154,20 @@ export function AssistantDetailsDropdownMenu({
                   close();
                 }}
               />
-              {showAssistantDetails ? (
+              {isShowAssistantDetails ? (
                 <DropdownMenu.Item
                   label={`More info`}
-                  onClick={(e) => {
-                    e.stopPropagation();
-                    close();
-                    showAssistantDetails();
+                  onClick={() => {
```

Comment:
you removed "e.stopPropagation()" I know i had some visual glitches because the click would sometime traverse to another assistant "button".

Reply:
actually the stopPropagation is preventing the menu to close, which creates other glitches . I'll dig into it

Reply:
I used to have a close() call as well, maybe it's enough to fix it ?

Reply:
seen irl, issue has been fixed in dropdown menu